guides/release/components/helper-functions.md

```diff
@@ -315,7 +315,69 @@ components.
 ```
 
 In the component's template, you can then use the `person` object:
-
 ```handlebars {data-filename=app/components/greeting/template.hbs}
 Hello, {{@person.givenName}} {{@person.familyName}}
 ```
+
+### The `in-element` helper
+
+Using the [`{{in-element}}`](https://api.emberjs.com/ember/3.27/classes/Ember.Templates.helpers/methods/in-element?anchor=in-element) helper, you can render content into a specific DOM element that is in a different part of the page. For instance, we might want
+to render a modal, tooltip or dropdown.
+
+Let's say I wanted to show a dropdown when a button is clicked on.
+
+Below, we have the button we want to click on, a div element that will hold the dropdown and the dropdown component. We pass in `showDropdown` that will add the dropdown into our div.
```

Comment:
I think the following sentence will sound more natural when read. What do you think?

```suggestion
Suppose we want to show a dropdown when the user clicks on a button. The code below shows a `<button>` element, a placeholder `<div>` element, and a dropdown component. The argument `@show`, when set to `true`, will add the dropdown to the placeholder div.
```

Reply:
Absolutely!